### DISCUSSION TITLE
Binary results

### DISCUSSION THREAD

**Discussion by crisarceramos** [dbeaver/dbeaver#182] [Q&A] [answered]
Hi dbeaver dev,
Not sure if this is an issue or I just don't know how to configure dbeaver since this is my first time using it. 
If I execute my query dbeaver displays binary data as binary. How can I make it display as text?
When I do the same query in SQLDeveloper I get a readable string representation of the binary data. I want dbeaver to do the same.

Thanks alot!

**Comment by serge-rider** [comment]
Could you clarify it a bit? What is the column type (VARBINARY, BLOB, .. ?).
Maybe some screenshot would help.

**Comment by r-richmond** [comment]
hey @crisarceramos,

you can format binary results under preferences>common>result sets>Binaries.

p.s. There is also a current request (#124) open to expand the number of binary formatting options. So feel free to express interest in that if the current binary formatting options aren't sufficient.

**Comment by serge-rider** [comment, marked as answer]
Yep. Binary formatting can be changed in preferences:
![binary-formatting](https://cloud.githubusercontent.com/assets/6398845/13009003/fd8a479a-d1aa-11e5-97a6-42aa87f61deb.png)
If you need something more sophisticated then this is duplicate of #124 indeed.
